ndk-patches/23c/stdio.h.patch

```diff
@@ -1,49 +1,53 @@
 --- ./usr/include/stdio.h.orig	2021-08-16 11:48:44.308954400 +0200
 +++ ./usr/include/stdio.h	2021-08-16 11:50:00.438924014 +0200
-@@ -44,6 +44,9 @@
+@@ -44,6 +44,11 @@
  #include <stdarg.h>
  #include <stddef.h>
  
++#if !defined(__swift__)
```

Comment:
Not needed, the Swift changes are mainly for compiling Swift source on a Termux device with the `ndk-sysroot` package. If you don't use Swift or compile on-device with this 23c sysroot, you don't need it.